nuxeo-retention-core/src/main/resources/schema/retention_rule.xsd

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<xs:schema targetNamespace="http://www.nuxeo.org/ecm/schemas/retention_rule/"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nxs="http://www.nuxeo.org/ecm/schemas/retention_rule/">
+
+  <xs:complexType name="retention_rule_condition">
+    <xs:sequence>
+      <xs:element name="expression" type="xs:string" />
+      <xs:element name="event" type="xs:string" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:element name="ruleId" type="xs:string" />
+  <xs:element name="beginCondition" type="nxs:retention_rule_condition" />
+  <xs:element name="endCondition" type="nxs:retention_rule_condition" />
+  <xs:element name="beginDelayInMillis" type="xs:long" />
```

Comment:
Maybe we can use this format that looks standard: https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-